wfl/cli/cli_options.py

```diff
@@ -1,12 +1,43 @@
-"""
-These are generalised cli options, merged decorators for repeated operations
-"""
+import click
+from wfl.configset import ConfigSet, OutputSpec
+from ase.io.extxyz import key_val_str_to_dict
 
-from click import STRING, argument, option
 
+def _to_ConfigSet(ctx, param, value):
+    return ConfigSet(value)
 
-def file_input_options(f):
-    """"""
-    f = argument("inputs", nargs=-1)(f)
-    f = option("--index", "-i", type=STRING, required=False, help="Pass this index to configset globally")(f)
+def inputs(f):
+    """Create ConfigSet for given filename(s)"""
+    f = click.option("--inputs", "-i", required=True, multiple=True, callback=_to_ConfigSet,
+                    help='Input xyz file(s) to create ConfigSet from')(f) 
     return f
+
+
+def _to_OutputSpec(ctx, param, value):
+    return OutputSpec(value)
+
+
+def outputs(f):
+    """Create OutputSpec for given filename"""
+    f = click.option('--outputs', '-o', required=True, callback=_to_OutputSpec,
+                     help="Ouput file to create OutputSpec from.")(f)
+    return f
+
+def _parse_extra_info(ctx, param, value):
+    if value is not None:
+        return key_val_str_to_dict(value)
+    else:
+        return {}
+
+def extra_info(f):
+    """Parse key=val string and return a dictionary"""
+    f = click.option("--extra-info", "-i", callback=_parse_extra_info, help="Extra key=val pairs to add to Atoms.info")(f)
+    return f
+
+def descriptor(f):
```

Comment:
Is there a use case for putting this in a centralized location, as opposed to right where the descriptor calc CLI is defined?  I don't obviously see what other command might need a standardized `--descriptor` flag.